config/routes.rb

```diff
@@ -0,0 +1,30 @@
+Rails.application.routes.draw do
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+  get 'reviews/index'
+  get 'ratings/index'
+  # get '/logged_in', to: 'sessions#is_logged_in?'
+  # post '/login', to: 'sessions#create'
+  # delete '/logout', to: 'sessions#destroy'
+  # get '/logged_in', to: 'sessions#is_logged_in?'
+  post '/login',    to: 'sessions#create'
+  post '/logout',   to: 'sessions#destroy'
+  get '/logged_in', to: 'sessions#islogged_in?'
+  resources :items
+  resources :reservations
```

Comment:
- [ ] In this case it's better to specify which actions you will be using here, so that useless routes aren't created.

Reply:
Done.